src/Cli/dotnet/CommonLocalizableStrings.resx

```diff
@@ -708,4 +708,10 @@ The default is 'true' if a runtime identifier is specified.</value>
   <data name="CorruptSolutionProjectFolderStructure" xml:space="preserve">
     <value>The solution file '{0}' is missing EndProject tags or has invalid child-parent project folder mappings around project GUID: '{1}'. Manually repair the solution or try to open and save it in Visual Studio."</value>
   </data>
-</root>
+  <data name="ArchArgumentName" xml:space="preserve">
+    <value>ARCH</value>
+  </data>
+  <data name="OsArgumentName" xml:space="preserve">
```

Comment:
convention for two letter abbr in PascalCase is `OSArgumentName`, `OSDescription`, `TargetOS` etc. starting with three letters, it uses the regular pacal casing `XmlSerializer`